package.json

```diff
@@ -36,34 +37,40 @@
     "react-dom": "18.3.1"
   },
   "devDependencies": {
+    "@eslint/compat": "^1.1.1",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.10.0",
```

Comment:
It may be better to leave it the default way as its more predictable and how most projects are structured.
This may not count but installing a new package re-sorts the dependencies alphabetically.